src/libraries/Microsoft.PowerFx.Core/Localization/Span.cs

```diff
@@ -60,8 +87,13 @@ public bool StartsWith(string script, string match)
             return Min + match.Length <= script.Length && string.CompareOrdinal(script, Min, match, 0, match.Length) == 0;
         }
 
-        // Generic span replacer. Given a set of unordered spans and replacement strings for
-        // each, this produces a new string with all the specified spans replaced accordingly.
+        /// <summary>
+        /// Generic span replacer. Given a set of unordered spans and replacement strings for 
+        /// each, this produces a new string with all the specified spans replaced accordingly.
+        /// </summary>
+        /// <param name="script"></param>
+        /// <param name="worklist"></param>
+        /// <returns></returns>
         [TransportDisabled]
         public static string ReplaceSpans(string script, IEnumerable<KeyValuePair<Span, string>> worklist)
```

Comment:
Not sure ReplaceSpans or StartsWith need to be public. They're internal utilities, but I don't think they need to be exposed.

Reply:
Same as above - this was already public. Will make internal.